index.js

```diff
@@ -1,8 +1,13 @@
 const axios = require('axios');
-const { ASTEROIDS_FEED_URL, API_KEY } = require('./environment');
+const moment = require('moment');
 
-const START_DATE = '2024-02-26';
-const END_DATE = '2024-03-01';
+const DATE_TEMPLATE = 'YYYY-MM-DD';
+
+const lastFriday = moment().day(-2);
```

Comment:
Hey, this will always return last Friday base on current date? Did I understand correctly?
day(0) = Su
day(1) = Mo
day(6) = Sa

Reply:
Yes, you understood correctly.
Some examples you can find here:
https://www.geeksforgeeks.org/moment-js-moment-day-method/